plugins/modules/keycloak_user.py

```diff
@@ -0,0 +1,496 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2019, INSPQ (@elfelip)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: keycloak_user
+short_description: create and Configure a user in Keycloak
+description:
+    - This module creates, removes or update Keycloak users.
+version_added: 7.0.0
+options:
+    realm:
+        description:
+            - The name of the realm in which is the client.
+        default: master
+        type: str
+    username:
+        description:
+            - username for the user.
```

Comment:
Paragraphs should always contain complete sentences.
```suggestion
            - Username for the user.
```